src/test/java/com/ezylang/evalex/parser/TokenizerStringLiteralTest.java

```diff
@@ -96,4 +97,20 @@ void testErrorUnmatchedQuoteOffset() {
         .isInstanceOf(ParseException.class)
         .hasMessage("Closing quote not found");
   }
+
+  @Test
+  void testSingleQuoteAllowed() throws ParseException {
+    assertThatThrownBy(() -> new Tokenizer("'hello'", configuration).parse())
+        .isInstanceOf(ParseException.class);
+
+    ExpressionConfiguration config =
+        ExpressionConfiguration.builder().singleQuoteStringLiteralsAllowed(true).build();
+
+    assertAllTokensParsedCorrectly(
```

Comment:
I would like to have more tests covering the behavior, if `singleQuoteStringLiteralsAllowed` is set to true:
- An equivalent of the `testErrorUnmatchedQuote...` tests here, using single quotes.
- Check if an escaped single quote is correctly parsed inside a single quote.
- Correct usage of single quotes inside double quotes and vice versa.